### PR TITLE
Update power-parity-pricing-strategies example

### PR DESCRIPTION
**Pull request by goncy**
Update power-parity-pricing-strategies example for middleware GA

### CODE REVIEW

File: edge-functions/power-parity-pricing-strategies/pages/_app.tsx [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 import type { AppProps } from 'next/app'
 import type { LayoutProps } from '@vercel/examples-ui/layout'
+import Image from 'next/image'
 import { getLayout, Text } from '@vercel/examples-ui'
-import '@vercel/examples-ui/globals.css'
+
 import map from '../public/map.svg'
 
+import '@vercel/examples-ui/globals.css'
+
```
Comment (lfades) [inline edge-functions/power-parity-pricing-strategies/pages/_app.tsx:9]: @goncy What are these changes for?

File: edge-functions/power-parity-pricing-strategies/middleware.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { NextRequest, NextResponse } from 'next/server'
+
+// Set pathname were middleware will be executed
+export const config = {
+  matcher: '/edge',
+}
+
+export function middleware(req: NextRequest) {
+  // Clone URL
+  const url = req.nextUrl.clone()
```
Comment (lfades) [inline edge-functions/power-parity-pricing-strategies/middleware.ts:10]: We don't have to clone the URL

File: edge-functions/power-parity-pricing-strategies/pages/_app.tsx (hunk 1 above)
Comment (goncy) [reply edge-functions/power-parity-pricing-strategies/pages/_app.tsx:9]: Just to have the types on top and the css on the bottom

**Review by dominiksipowicz — APPROVED** [review]

**Review by mistressxalexis — COMMENTED** [review]
Fíni

**Pull request merged**